props/saber_BC_buttons.h

```diff
@@ -1253,9 +1270,13 @@ struct BCVolumeMode : public SPEC::SteppedMode {
       if (current_volume_ >= max_volume_) {
         current_volume_ = max_volume_;
         QuickMaxVolume();
-      } else {
+      } else if (millis() - trigger_time_ > 100 && !sound_library_.busy()) {
+        // If anything is playing or 100ms hasn't passed, wait to queue SayVolumeUp
```

Comment:
This comment doesn't seem to make sense. I mean, if we are here in the code, then nothing is playing and 100ms has indeed passed...